src/resolvers/resolver.ts

```diff
@@ -12,20 +15,36 @@ const resolvers = {
       const profiles = await Profile.find({});
       return profiles;
     },
+
+    async getAllCohorts() {
+      const allCohorts = await Cohort.find({}).populate('members').populate("coordinator");
+      return allCohorts;
+    },
+
+    async getCohort(_:any, { cohortName }: any) {
+      const getCohort = await Cohort.findOne({ name:cohortName }).populate('members').populate("coordinator");
```

Comment:
Can we prefer to find by id?
If we are using the name, remember to normalize the name before searching and saving to the database. 
Example, we can stick with just `lowercase` and do the transformation on the frontend. Just make sure to convert the user input to lowercase before searching or use `regexmatch` of mongoose

```
Cohort.find({ $in: [
      { name: { $regex: cohortName, $options: "i" } },
    ], })
// Note that this would also include partials search and would return an array
// which makes "id" a better preference
```